services/QuillLMS/app/controllers/canvas_integration/lti_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module CanvasIntegration
+  class LtiController < ApplicationController
+    # launch is loaded within Canvas in an iframe muddling the CSRF token
+    skip_before_action :verify_authenticity_token, only: :launch
+
+    SSO_LINK_TEXT = 'This link will redirect you to Quill.org for SSO'
```

Comment:
Is this the right text for this link?  I imagine that most normal users won't know what "SSO" means.  Maybe not a big deal, but I'm always curious about customer-facing copy.

Reply:
This is definitely placeholder copy, but let's make it closer to what it will be.
```suggestion
    SSO_LINK_TEXT = 'Log in to Quill.org with your Canvas credentials'
```